src/functions/createSwapProposal/schema.ts

```diff
@@ -5,6 +5,10 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+/**
+ * Schema in the `json-schema-to-ts` format
+ * @see https://www.npmjs.com/package/json-schema-to-ts
```

Comment:
It could be hard to find context if there was any need to change this schema, so I added the reference rules.